README.adoc

```diff
@@ -5842,6 +5842,21 @@ config
 
 This is simpler and more efficient.
 
+=== Don't Extend `Data.define` [[no-extend-data-define]]
+
+Don't extend an instance initialized by `Data.define`.
+Extending it introduces a superfluous class level and may also introduce weird errors if the file is required multiple times.
```

Comment:
> introduce weird errors if the file is required multiple times

I don't really get this part.

```ruby
class A < Data.define(:x); end
class A < Data.define(:x); end # => TypeError: superclass mismatch for class A
class B < Struct.new(:x); end
class B < Struct.new(:x); end # => TypeError: superclass mismatch for class B
```

Also, `require` docs:

> Loads the given name, returning true if successful and **false** if the feature is already loaded.

Reply:
Nice catch! I think I must've forgot to remote the part about _introduce weird errors..._ when I copied and adjusted the description from the `[[no-extend-struct-new]]`. 

I guess now the question is - what did the author of `[[no-extend-struct-new]]` meant there by _weird errors_ and if it's still relevant 🤔 

I'll remove the part after _and_, thank you!